index.html

```diff
@@ -1,86 +1,162 @@
+                </div>
+                <div class="sliderPost">
+                    <div class="sliderPostImg">
+                        <img src="image/slider_2.png" alt="sliderPostImg">
+                    </div>
+                    <div class="sliderPostMain">
+                        <a href="" class="sliderPostTitle">Lorem ipsum dolor sit amet consectetur</a>
+                        <a href="" class="sliderPostCategory">Chuyện của tôi</a>
+                    </div>
+                </div>
+                <div class="sliderPost">
+                    <div class="sliderPostImg">
+                        <img src="image/slider_3.png" alt="sliderPostImg">
+                    </div>
+                    <div class="sliderPostMain">
+                        <a href="" class="sliderPostTitle">Lorem ipsum dolor sit amet consectetur</a>
+                        <a href="" class="sliderPostCategory">Chuyện của tôi</a>
+                    </div>
+                </div>
+            </div>
+            <div class="swipe">
+                <span class="arrow arrowLeft">
+                    <img src="image/Arrow 1.png" alt="arrowSwipe">
+                </span>
+                <span class="swipeText">Swipe to discover more <span style="color: orange;">popular</span> stories</span>
+                <span class="arrow arrowRight">
+                    <img src="image/Arrow 1.png" alt="arrowSwipe">
+                </span>
             </div>
         </div>
-      </div>
-    </div>
-  </body>
+    </header>
+    
+    <main>
+        <section class="postSection">
+            <div class="container postContain">
+                <div class="txtTitle">
+                    <span>All stories</span>
+                    <div class="arrowStories">
+                        <img src="image/Arrow 1.png" alt="ArrowStories">
+                    </div>
+                </div>
+                <div class="post">
+                    <div class="postImg">
+                        <img src="image/post_1.png" alt="postImg">
+                    </div>
+                    <div class="postMain">
+                        <a href="" class="postCategory">Chuyện của tôi</a>
+                        <a href="" class="postTitle">Click this card for redirect to full story</a>
+                        <div class="postDescription">Lorem ipsum dolor sit amet consectetur adipiscing elit Lorem ipsum dolor sit amet consectetur adipiscing elit. ipsum dolor sit amet consectetur adipiscing elit.</div>
+                        <div class="postInfoShare">
+                            <p class="postInfo">bởi <a href="" style="font-weight: 700;">Pikachu</a>, 07/07/2011</p>
+                            <a href="" class="postShare">Share</a>
+                        </div>
+                    </div>
+                </div>
+                <div class="post">
+                    <div class="postImg">
+                        <img src="image/post_2.png" alt="postImg">
```

Comment:
What's alt attribute for? **alt** means if the image loads failed, it will note the image as "postImg", right?
Don't you think it's a little vague to the user?
Any idea to change this alt to better description for the story (post) image?

Reply:
@nguyenvl179 You haven't resolved this. If you have questions about anything, feel free to discuss in this conversation.